apps/modernization-api/src/test/java/gov/cdc/nbs/patient/PatientMother.java

```diff
@@ -11,37 +12,43 @@
 @Component
 public class PatientMother {
 
-  private final MotherSettings settings;
-  private final TestUniqueIdGenerator idGenerator;
-  private final EntityManager entityManager;
-  private final TestPatients patients;
-  private final TestPatientCleaner cleaner;
-
-  public PatientMother(
-      final MotherSettings settings,
-      final TestUniqueIdGenerator idGenerator,
-      final EntityManager entityManager,
-      final TestPatients patients,
-      final TestPatientCleaner cleaner
-  ) {
-    this.settings = settings;
-    this.idGenerator = idGenerator;
-    this.entityManager = entityManager;
-    this.patients = patients;
-    this.cleaner = cleaner;
-  }
-
-  void reset() {
-    this.cleaner.clean(settings.starting());
-    this.patients.reset();
-  }
-
-  public Person patient() {
-    Person patient = PersonMother.generateRandomPerson(idGenerator.next());
-
-    this.entityManager.persist(patient);
-
-    patients.available(patient.getId());
-    return patient;
-  }
+    private final MotherSettings settings;
+    private final TestUniqueIdGenerator idGenerator;
+    private final PatientLocalIdentifierGenerator localIdentifierGenerator;
+    private final EntityManager entityManager;
+    private final TestPatients patients;
+    private final TestPatientCleaner cleaner;
+
+    public PatientMother(
+        final MotherSettings settings,
+        final TestUniqueIdGenerator idGenerator,
+        final PatientLocalIdentifierGenerator localIdentifierGenerator,
+        final EntityManager entityManager,
+        final TestPatients patients,
+        final TestPatientCleaner cleaner
+    ) {
+        this.settings = settings;
+        this.idGenerator = idGenerator;
+        this.localIdentifierGenerator = localIdentifierGenerator;
+        this.entityManager = entityManager;
+        this.patients = patients;
+        this.cleaner = cleaner;
+    }
+
+    void reset() {
+        this.cleaner.clean(settings.starting());
+        this.patients.reset();
+    }
+
+    public Person patient() {
```

Comment:
maybe this could be named `createPatient` or something, to better illustrate that it is being persisted.